packages/rpc-types/src/account-filters.ts

```diff
@@ -1,26 +1,43 @@
+import { Base58EncodedBytes, Base64EncodedBytes } from './encoded-bytes';
+
 export type DataSlice = Readonly<{
     /** The number of bytes to return */
     length: number;
     /** The byte offset from which to start reading */
     offset: number;
 }>;
 
+type ProgramNotificationsMemcmpFilterBase58 = Readonly<{
+    /**
+     * The bytes to match, as a base-58 encoded string.
+     *
+     * Data is limited to a maximum of 128 decoded bytes.
+     */
+    bytes: Base58EncodedBytes;
+    /** The encoding to use when decoding the supplied byte string */
+    encoding: 'base58';
+    /** The byte offset into the account data from which to start the comparison */
+    offset: bigint;
+}>;
+
+type ProgramNotificationsMemcmpFilterBase64 = Readonly<{
+    /**
+     * The bytes to match, as a base-64 encoded string.
+     *
+     * Data is limited to a maximum of 128 decoded bytes.
```

Comment:
The Intellisense will now change the comment on this field depending on what you supplied for `encoding`.

Reply:
Is base64 limited to 128 bytes as well here? Usually that's just base58 AFAIK

Reply:
It is! https://github.com/steveluscher/agave/blob/dc03d30163c39a235216244f1dcc17db5aab769e/rpc-client-types/src/filter.rs#L45-L49